browser/index.js

```diff
@@ -3,25 +3,40 @@ const version = '__VERSION__'
 const url = 'https://github.com/bugsnag/bugsnag-js'
 
 const Client = require('../base/client')
-const { map, reduce, keys } = require('../base/lib/es-utils')
+const { map, reduce } = require('../base/lib/es-utils')
 
 // extend the base config schema with some browser-specific options
 const schema = { ...require('../base/config').schema, ...require('./config') }
 
-const plugins = {
-  'window onerror': require('./plugins/window-onerror'),
-  'unhandled rejection': require('./plugins/unhandled-rejection'),
-  'device': require('./plugins/device'),
-  'context': require('./plugins/context'),
-  'request': require('./plugins/request'),
-  'throttle': require('../base/plugins/throttle'),
-  'console breadcrumbs': require('./plugins/console-breadcrumbs'),
-  'navigation breadcrumbs': require('./plugins/navigation-breadcrumbs'),
-  'interaction breadcrumbs': require('./plugins/interaction-breadcrumbs'),
-  'inline script content': require('./plugins/inline-script-content'),
-  'sessions': require('./plugins/sessions'),
-  'ip': require('./plugins/ip')
-}
+const pluginWindowOnerror = require('./plugins/window-onerror')
+const pluginUnhandledRejection = require('./plugins/unhandled-rejection')
+const pluginDevice = require('./plugins/device')
+const pluginContext = require('./plugins/context')
+const pluginRequest = require('./plugins/request')
+const pluginThrottle = require('../base/plugins/throttle')
+const pluginConsoleBreadcrumbs = require('./plugins/console-breadcrumbs')
+const pluginNavigationBreadcrumbs = require('./plugins/navigation-breadcrumbs')
+const pluginInteractionBreadcrumbs = require('./plugins/interaction-breadcrumbs')
+const pluginInlineScriptContent = require('./plugins/inline-script-content')
+const pluginSessions = require('./plugins/sessions')
+const pluginIp = require('./plugins/ip')
+const pluginStripQueryString = require('./plugins/strip-query-string')
```

Comment:
This seems like a sensible change - plugins had names purely for the sake of fetching them out by name.

Reply:
Yeah, they needed to be accessible by name and also enumerable. But the pretty object keys wouldn't get compressed, so I realised I could just define them all as variables and then stick them in an array for better results.